packages/ckeditor5-utils/src/dom/resizeobserver.js

```diff
@@ -1,378 +0,0 @@
-
-		// Remove the element callback. Check if exist first in case someone
-		// called destroy() twice.
-		if ( callbacks ) {
-			callbacks.delete( callback );
-
-			// If no callbacks left for the element, also remove the element.
-			if ( !callbacks.size ) {
-				ResizeObserver._elementCallbacks.delete( element );
-				ResizeObserver._observerInstance.unobserve( element );
-			}
-		}
-
-		if ( ResizeObserver._elementCallbacks && !ResizeObserver._elementCallbacks.size ) {
-			ResizeObserver._observerInstance = null;
-			ResizeObserver._elementCallbacks = null;
-		}
-	}
-
-	/**
-	 * Returns are registered resize callbacks for the DOM element.
-	 *
-	 * @private
-	 * @static
-	 * @param {HTMLElement} element
-	 * @returns {Set.<HTMLElement>|null}
-	 */
-	static _getElementCallbacks( element ) {
-		if ( !ResizeObserver._elementCallbacks ) {
-			return null;
-		}
-
-		return ResizeObserver._elementCallbacks.get( element );
-	}
-
-	/**
-	 * Creates the single native observer shared across all `ResizeObserver` instances.
-	 * If the browser does not support the native API, it creates a polyfill.
-	 *
-	 * @private
-	 * @static
-	 */
-	static _createObserver() {
-		let ObserverConstructor;
-
-		// TODO: One day, the `ResizeObserver` API will be supported in all modern web browsers.
-		// When it happens, this module will no longer make sense and should be removed and
-		// the native implementation should be used across the project to save bytes.
-		// Check out https://caniuse.com/#feat=resizeobserver.
-		if ( typeof global.window.ResizeObserver === 'function' ) {
-			ObserverConstructor = global.window.ResizeObserver;
-		} else {
-			ObserverConstructor = ResizeObserverPolyfill;
-		}
-
-		ResizeObserver._observerInstance = new ObserverConstructor( entries => {
-			for ( const entry of entries ) {
-				const callbacks = ResizeObserver._getElementCallbacks( entry.target );
-
-				if ( callbacks ) {
-					for ( const callback of callbacks ) {
-						callback( entry );
-					}
-				}
-			}
-		} );
-	}
-}
-
-/**
- * The single native observer instance (or polyfill in browsers that do not support the API)
- * shared across all {@link module:utils/dom/resizeobserver~ResizeObserver} instances.
- *
- * @static
- * @protected
- * @readonly
- * @property {Object|null} module:utils/dom/resizeobserver~ResizeObserver#_observerInstance
- */
-ResizeObserver._observerInstance = null;
-
-/**
- * A mapping of native DOM elements and their callbacks shared across all
- * {@link module:utils/dom/resizeobserver~ResizeObserver} instances.
- *
- * @static
- * @private
- * @readonly
- * @property {Map.<HTMLElement,Set>|null} module:utils/dom/resizeobserver~ResizeObserver#_elementCallbacks
- */
-ResizeObserver._elementCallbacks = null;
-
-/**
- * A polyfill class for the native [`ResizeObserver`](https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver).
- *
- * @private
- * @mixes module:utils/domemittermixin~DomEmitterMixin
- */
-class ResizeObserverPolyfill {
```

Comment:
Why did you remove this file completely? We were expected only to remove the polyfill AFAIK.

Reply:
Ugh, it's below... diff tricked me :D